Formula/aws-vault.rb

```diff
@@ -0,0 +1,26 @@
+class AwsVault < Formula
+  desc "Securely store and access AWS credentials in development environments"
+  homepage "https://github.com/99designs/aws-vault"
+  url "https://github.com/99designs/aws-vault/archive/v6.3.1.tar.gz"
+  sha256 "433df90b7ed1cf1ec08aa75a4f1f993edfe5fa3fecfff5519574613ab0ab4630"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    os = "darwin"
+    on_linux { os = "linux" }
+    arch = Hardware::CPU.arm? ? "arm64" : "amd64"
+
+    system "make", "aws-vault-#{os}-#{arch}"
+    system "make", "install", "INSTALL_DIR=#{bin}"
```

Comment:
```suggestion
    system "make", "aws-vault-#{os}-#{arch}", "VERSION=#{version}-homebrew"
    system "make", "install", "INSTALL_DIR=#{bin}", "VERSION=#{version}-homebrew"
```

This gets rid of the "Not a git repository" errors, properly sets the version, and includes something in the version string to differentiate homebrew builds from upstream.

Reply:
Let's do `VERSION=#{version}-#{tap.user}` in case users extract this formula to their own taps.